src/proposals/20240110/SparkEthereum_20240110.t.sol

```diff
@@ -0,0 +1,637 @@
+    address constant REWARDS_OPERATOR      = 0x8076807464DaC94Ac8Aa1f7aF31b58F73bD88A27;
+    address constant TRANSFER_STRATEGY     = 0x11aAC1cA5822cf8Ba6d06B0d84901940c0EE36d8;
+    address constant WETH_ATOKEN           = 0x59cD1C87501baa753d0B5B5Ab5D8416A45cD71DB;
+
+    address constant DAI    = 0x6B175474E89094C44Da98b954EedeAC495271d0F;
+    address constant GNO    = 0x6810e776880C02933D47DB1b9fc05908e5386b96;
+    address constant RETH   = 0xae78736Cd615f374D3085123A210448E74Fc6393;
+    address constant SDAI   = 0x83F20F44975D03b1b09e64809B757c47f942BEeA;
+    address constant USDC   = 0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48;
+    address constant USDT   = 0xdAC17F958D2ee523a2206206994597C13D831ec7;
+    address constant WBTC   = 0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599;
+    address constant WETH   = 0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2;
+    address constant WSTETH = 0x7f39C581F595B53c5cb19bD0b3f8dA6c935E2Ca0;
+
+    address public constant DAI_ORACLE_OLD    = 0xAed0c38402a5d19df6E4c03F4E2DceD6e29c1ee9;
+    address public constant DAI_ORACLE_NEW    = 0x42a03F81dd8A1cEcD746dc262e4d1CD9fD39F777;
+    address public constant WSTETH_ORACLE_OLD = 0xA9F30e6ED4098e9439B2ac8aEA2d3fc26BcEbb45;
+    address public constant WSTETH_ORACLE_NEW = 0x8B6851156023f4f5A66F68BEA80851c3D905Ac93;
+
+    address constant WHALE1   = 0xf8dE75c7B95edB6f1E639751318f117663021Cf0;
+    address constant WHALE2   = 0xAA1582084c4f588eF9BE86F5eA1a919F86A3eE57;
+    address constant GNO_USER = 0xe1d0508d4976Bd4b8552fBe5c31Cc0F023258f0C;  // Has a small GNO position
+
+    address constant CURRENT_HAT = 0x4F09EbaA1A5e52EB95c97f3b9fa3fb398D004698;  // Happens to be the last passed spell
+
+    ISparkLendFreezerMom        freezerMom           = ISparkLendFreezerMom(FREEZER_MOM);
+    IEACAggregatorProxy         daiOracle            = IEACAggregatorProxy(DAI_ORACLE_NEW);
+    PullRewardsTransferStrategy rewardStrategy       = PullRewardsTransferStrategy(TRANSFER_STRATEGY);
+    IAToken                     wethAToken           = IAToken(WETH_ATOKEN);
+    IERC20                      wsteth               = IERC20(WSTETH);
+    IRewardsController          incentivesController = IRewardsController(INCENTIVES_CONTROLLER);
+
+    uint256 REWARD_AMOUNT = 20 ether;
+    uint256 DURATION      = 30 days;
+    uint256 STETH_INDEX   = 1.1525e18; // Approximate conversion rate for approx APY calcs
+
+    address claimAddress1 = makeAddr("claimAddress1");
+    address claimAddress2 = makeAddr("claimAddress2");
+
+    constructor() {
+        id = '20240110';
+    }
+
+    function setUp() public {
+        vm.createSelectFork(getChain('mainnet').rpcUrl, 18934612);  // Jan 4, 2024
+        payload = deployPayload();
+
+        loadPoolContext(poolAddressesProviderRegistry.getAddressesProvidersList()[0]);
+    }
+
+    // --- Configuration Changes ---
+
+    function testFreezerMomDeploy() public {
```

Comment:
One and the only one style nitpick - I'd rename all `testDescriptionOfTheTest` named tests to `test_descriptionOfTheTest`

Reply:
Added this in a PR. You can merge it, or close it, your call.
https://github.com/marsfoundation/spark-spells/pull/30